packages/components/src/circular-option-picker/style.scss

```diff
@@ -4,6 +4,7 @@ $color-palette-circle-spacing: 12px;
 .components-circular-option-picker {
 	display: inline-block;
 	width: 100%;
+	min-width: 244px;
```

Comment:
I can confirm this fixes things, but I wonder where `244px` came from?

Reply:
I'm not sure we have a set width for these... they change from one to another. So with this one, I made the width 244px + the additional popover padding which equates to 280px. And 280px is the same as the Sidebar Settings.

Reply:
Thats probably fine; It fixes the immediate problem.

That said, I worry this might look strange if there's only 2-3 color swatches; It could lead to some awkward whitespace.